semantic-core/src/Data/Core/Parser.hs

```diff
@@ -0,0 +1,118 @@
+{-# LANGUAGE ExplicitForAll, FlexibleInstances, LambdaCase, MultiParamTypeClasses, OverloadedLists,
+             ScopedTypeVariables #-}
```

Comment:
I don’t think we’re using any of these extensions any more. No `forall`s, no instances, no `\case`, and the `HashSet` is being defined in `Data.Name` now.

Reply:
Fixed.